.palantir/revapi.yml

```diff
@@ -1056,6 +1056,9 @@ acceptedBreaks:
     - code: "java.method.removed"
       old: "method org.apache.iceberg.DataFiles.Builder org.apache.iceberg.DataFiles.Builder::withEqualityFieldIds(java.util.List<java.lang.Integer>)"
       justification: "Deprecations for 1.6.0 release"
+    - code: "java.method.removed"
```

Comment:
This is because I kept @jackye1995 commit.

Reply:
Also, the CI build si failing with other revapi issues (mostly source compatibility issues in fact), but I don't have issues when running revapi locally 🤷‍♂️

Reply:
If you fetch tags locally probably it should fail locally aswell. 

probably because new release tag added and we didn't update the master branch code about base version. 
https://github.com/apache/iceberg/pull/10752#issuecomment-2245143821

Reply:
So, basically we need to move this new addition under 1.6.0 not 1.5.0

Reply:
I fetched the tags, and it still doesn't fail 😆  Should I add revapi exceptions?

Reply:
I think we can try this
1. Merge https://github.com/apache/iceberg/pull/10754
2. Remove the current addition in the file `revapi.yml` as it is under 1.5.0
3. freshly run `revapiAcceptBreak`, so it will add it under 1.6.0

If this doesn't work. We need to debug why old version is coming as `none`

Reply:
For now I added the missing exceptions, we can revisit this later when #10754 is merged.

Reply:
That PR is merged. You can rebase and try now.

Reply:
Thank you @ajantha-bhat! I was able to remove all the exceptions. All good 👍

Reply:
I _think_ I found the issue I was having with revapi, my last commit should hopefully solve this 🤞